### PR TITLE
Adding options

### PR DESCRIPTION
**Pull request by maddhruv**
### Done - 
- `open`
- `not-random`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ good-first-issue <project>
 #### Options
 
 - `-o, --open` - open in browser
+- `--not-random` - Not random, return the top issue
```
Comment (bnb) [inline README.md]: I prefer `-f, --first` – seems to make more sense in this context.

File: bin/good-first-issue.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ cli
   .description('CLI tool to find good first issues.')
   .arguments('[project]')
   .option('-o --open', 'Open in browser')
+  .option('--not-random', 'Not random, return the top issue')
```
Comment (bnb) [inline bin/good-first-issue.js]: See previous comment about `-f, --first`

**Comment by maddhruv** [comment]
please check again :)

**Comment by bnb** [comment]
LGTM :shipit:

**Pull request merged**